source/languages.ttl

```diff
@@ -0,0 +1,3171 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+@base <https://id.kb.se/language/> .
+
+<9fd> a schema:Language ;
+    rdfs:comment "-1500 (Librisdefinierad kod)"@sv ;
+    skos:notation "9fd",
+        "9fd"^^kbv:LibrisMarcLanguageCode ;
+    skos:prefLabel "Forndanska"@sv .
+
+<9fs> a schema:Language ;
+    rdfs:comment "-1500 (Librisdefinierad kod)"@sv ;
+    skos:notation "9fs",
+        "9fs"^^kbv:LibrisMarcLanguageCode ;
+    skos:prefLabel "Fornsvenska"@sv .
+
+<9so> a schema:Language ; 
+    rdfs:comment "Obsolet Librisdefinierad kod: använd \"smn\" (enaresamiska), \"sms\" (skoltsamiska) eller \"smi\" (samiska)"@sv ;
+    dc:isReplacedBy <smn>, <sms>, <smi> ;
+    skos:notation "9so",
+        "9so"^^kbv:LibrisMarcLanguageCode ;
+    skos:prefLabel "Östsamiska"@sv .
+
+<aar> a schema:Language ;
+    skos:altLabel "Afariska"@sv ;
+    skos:notation "aar",
+        "aar"^^kbv:ISO639-2,
+        "aar"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Afar"@sv .
+
+<abk> a schema:Language ;
+    skos:notation "abk",
+        "abk"^^kbv:ISO639-2,
+        "abk"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Abchaziska"@sv .
+
+<ace> a schema:Language ;
+    skos:altLabel "Atjehnesiska"@sv ;
+    skos:notation "ace",
+        "ace"^^kbv:ISO639-2,
+        "ace"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Achenesiska"@sv .
+
+<ach> a schema:Language ;
+    skos:notation "ach",
+        "ach"^^kbv:ISO639-2,
+        "acb"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Acholi"@sv .
+
+<ada> a schema:Language ;
+    skos:notation "ada",
+        "ada"^^kbv:ISO639-2,
+        "ada"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Adangme"@sv .
+
+<ady> a schema:Language ;
+    skos:notation "ady",
+        "ady"^^kbv:ISO639-2,
+        "ady"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Adygeiska"@sv .
+
+<afa> a schema:Language ;
+    skos:notation "afa",
+        "afa"^^kbv:ISO639-2,
+        "afa"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Afro-asiatiska språk (övriga)"@sv .
+
+<afh> a schema:Language ;
+    rdfs:comment "Konstgjort språk"@sv ;
+    skos:notation "afh",
+        "afh"^^kbv:ISO639-2,
+        "afh"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Afrihili"@sv .
+
+<afr> a schema:Language ;
+    skos:notation "afr",
+        "afr"^^kbv:ISO639-2,
+        "afr"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Afrikaans"@sv .
+
+<ain> a schema:Language ;
+    skos:notation "ain",
+        "ain"^^kbv:ISO639-2,
+        "ain"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Ainu"@sv .
+
+<aka> a schema:Language ;
+    skos:notation "aka",
+        "aka"^^kbv:ISO639-2,
+        "aka"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Akan"@sv .
+
+<akk> a schema:Language ;
+    skos:notation "akk",
+        "akk"^^kbv:ISO639-2,
+        "akk"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Akkadiska"@sv .
+
+<alb> a schema:Language ;
+    skos:notation "alb",
+        "alb"^^kbv:ISO639-2,
```

Comment:
```suggestion
        "alb"^^kbv:ISO639-2,
        "sqi"^^kbv:ISO639-2,
```

Reply:
ISO-639-2 = ISO-639-2/B ∪ ISO-639-2/T so both B and T should exist as `^^kbv:ISO639-2`